drivers/digital-io/max149x6/iio_max14906.c

```diff
@@ -255,11 +255,11 @@ static int max14906_iio_read_do_mode(void *dev, char *buf, uint32_t len,
 				     const struct iio_ch_info *channel, intptr_t priv)
 {
 	struct max14906_iio_desc *iio_desc = dev;
-	struct max14906_desc *desc = iio_desc->max14906_desc;
```

Comment:
that commit title is too long, you don't need to put the whole file hierarchy in the commit title, just put the chip name and it's clear enough

if you want to get the files changed by a commit, it's easily doable with `git show [hash] --name-only`

Reply:
Making the commit title shorter.